source/common/ssl/utility.cc

```diff
@@ -65,7 +75,7 @@ std::string Utility::getSubjectFromCertificate(X509& cert) {
   return std::string(reinterpret_cast<const char*>(data), data_len);
 }
 
-int32_t Utility::getDaysUntilExpiration(X509* cert, TimeSource& time_source) {
+int32_t Utility::getDaysUntilExpiration(const X509* cert, TimeSource& time_source) {
```

Comment:
Envoy style suggests `const X509& cert`, i.e. reference style.

Reply:
I have changed this to reference style for other two functions added in this PR. But this one, I left as is because there is some special handling inside when it is nullptr and do not want to change it to handle this at call sites.